documentation/asciidoc/computers/configuration/device-tree.adoc

```diff
@@ -980,7 +980,7 @@ This sets an (exclusive) limit to the loaded device tree. By default the device
 If non-zero, turn on some extra logging for the firmware's device tree processing.
 
 `enable_uart`
-Enable the primary/console xref:configuration.adoc#configuring-uarts[UART] (ttyS0 on a Raspberry Pi 3, ttyAMA0 otherwise - unless swapped with an overlay such as miniuart-bt). If the primary UART is ttyAMA0 then enable_uart defaults to 1 (enabled), otherwise it defaults to 0 (disabled). This is because it is necessary to stop the core frequency from changing which would make ttyS0 unusable, so `enable_uart=1` implies core_freq=250 (unless force_turbo=1). In some cases this is a performance hit, so it is off by default. 
+Enable the primary/console xref:configuration.adoc#configuring-uarts[UART] (ttyS0 on a Raspberry Pi 3/4 or Zero W, ttyAMA0 otherwise - unless swapped with an overlay such as miniuart-bt). If the primary UART is ttyAMA0 then enable_uart defaults to 1 (enabled), otherwise it defaults to 0 (disabled). This is because it is necessary to stop the core frequency from changing which would make ttyS0 unusable, so `enable_uart=1` implies core_freq=250 (unless force_turbo=1). In some cases this is a performance hit, so it is off by default. 
```

Comment:
```suggestion
Enable the primary/console xref:configuration.adoc#configuring-uarts[UART] (ttyS0 on a Raspberry Pi 3, 4 and Zero W, ttyAMA0 otherwise - unless swapped with an overlay such as miniuart-bt). If the primary UART is ttyAMA0 then enable_uart defaults to 1 (enabled), otherwise it defaults to 0 (disabled). This is because it is necessary to stop the core frequency from changing which would make ttyS0 unusable, so `enable_uart=1` implies core_freq=250 (unless force_turbo=1). In some cases this is a performance hit, so it is off by default. 
```

Reply:
I guess this is probably all Raspberry Pis with a Wifi / Bluetooth chip, so also Raspberry Pi 400 and Raspberry Pi Zero W ?

Reply:
Zero W is in the list, but probably also the new Zero 2 W? The list is from uart.adoc, the 400 and Zero 2 W are missing there too.

Reply:
Yes, finger-slip, I meant to write Raspberry Pi Zero 2 W :rofl:

Reply:
@agners You're more than welcome to add 400 and Zero 2 W to `uart.adoc` too :slightly_smiling_face:

Reply:
I'll leave the `uart.adoc` part for someone else to tackle :smile: 

```suggestion
Enable the primary/console xref:configuration.adoc#configuring-uarts[UART] (ttyS0 on a Raspberry Pi 3, 4, 400, Zero W and Zero 2 W, ttyAMA0 otherwise - unless swapped with an overlay such as miniuart-bt). If the primary UART is ttyAMA0 then enable_uart defaults to 1 (enabled), otherwise it defaults to 0 (disabled). This is because it is necessary to stop the core frequency from changing which would make ttyS0 unusable, so `enable_uart=1` implies core_freq=250 (unless force_turbo=1). In some cases this is a performance hit, so it is off by default. 
```

Reply:
Yeah lets try and avoid huge lists of models if we can help it. If it's the same for all models with a wireless chip that's a good enough differentiator to use.